sdcm/utils/database_query_utils.py

```diff
@@ -190,6 +190,11 @@ def get_table_clustering_order(ks_cf: str, ck_name: str, session) -> str:
     return clustering_order
 
 
+def is_system_keyspace(keyspace: str) -> bool:
```

Comment:
how this change is related to this PR ?

I would expect it to be at the minimum on it's own commit